AUTHORS

```diff
@@ -142,6 +148,7 @@ Joydeep Mukherjee <joydeep1701@gmail.com>
 Justin Du <justin.d128@gmail.com>
 Jérôme <zolk232@gmail.com>
 K.N. Anantha Nandanan <ananthanandanan@gmail.com>
+Kajol <kajolkumarisingh222@gmail.com>
```

Comment:
Just as an FYI, she has signed the CLA with the `Kajol Kumari` name. (not sure which name we use here!)

Reply:
Better to use that then, updated.